modules/cardano.nix

```diff
@@ -57,6 +57,26 @@ globals: imports: params:
     ${concatStringsSep "\n" (map (host: "${toString host.ip} ${host.name}.cardano") hostList)}
     '';
 
+    services.dd-agent.processConfig = ''
+    init_config:
+
+    instances:
+    '' +
+    (if params.typeIsExplorer
+    then ''
+    - name: cardano-explorer
+      search_string: ['cardano-explorer']
+      ''
+    else ''
+    - name: cardano-node-simple
+      search_string: ['cardano-node-simple']
+      '') + ''
+    # nix string indentation hack (see DEVOPS-341)
+      exact_match: True
+      thresholds:
+        critical: [1, 1]
+    '';
+
```

Comment:
Was this copied in from another branch? Were there any changes made to from the original?

I recall working on this code to add the indentation hack. I know the indentation hack was verified in some environment like staging or TN06.

Reply:
Oh, right, it's being moved from `modules/cardano-staging.nix` with two changes:

* `exact_match` is enabled
* `cardano-node-simple` instead of `cardano-node` for the search string

Reply:
String concatenation should be converted to anitquoting here `${if ... then .. else}` to prevent any errors in identation.

Reply:
Done.